target-app/Networking/Services/TargetServices.swift

```diff
@@ -24,6 +24,12 @@ protocol TargetServicesProtocol {
 
 final class TargetServices: TargetServicesProtocol {
     
+    let apiClient: APIClient
+    
+    init(apiClient: APIClient) {
```

Comment:
Wdyt about set the BaseAPIClient.default as a default parameter? An then you can omit in the TargetServices initialisation
```suggestion
    init(apiClient: APIClient = BaseAPIClient.default) {
```

Reply:
oh right! thanks